Storage/Rust/RustPlaces.swift

```diff
@@ -269,15 +264,9 @@ public class RustPlaces {
         return error
     }
 
-    public func interrupt() {
```

Comment:
I haven't rebased from `main` in a bit but I needed to pull in [these changes](https://github.com/mozilla-mobile/firefox-ios/pull/10297/files#diff-4831bd77ff9fc5a2aad43d89be90fce6becbde4bd71db26792b2d373c2f57b1e) because when I created my PR, my local AS version was newer than the version iOS was using. Long story short, once I rebase no changes to this file will be included in this PR.